docs/shared/register-federated-cli.mdx

```diff
@@ -1,53 +1,42 @@
+
+    * The name that uniquely identifies the subgraph within your graph (e.g., `products`).
+    * The URL that your gateway will use to communicate with the subgraph (e.g., `http://products-graphql.svc.cluster.local:4001/`).
+    * The service's schema. The Rover CLI can either:
         * Use a `.gql` or `.graphql` file saved on your local machine, or
         * Perform an introspection query on the running service to fetch the schema
 
-2. Run the `apollo service:push` command, providing the following options (documented below):
+2. Run the `rover subgraph publish` command, providing your schema in one of the ways shown:
 
     ```bash
-    apollo service:push \
-      --localSchemaFile=./schema.gql \ # (or --endpoint)
-      --key=service:docs-example-graph:NYKgCqwfCyYPIm84WVXCdw
-      --graph=docs-example-graph \
-      --variant=local-development \
-      --serviceName=products \
-      --serviceURL=http://products-graphql.svc.cluster.local:4001/
+    # Provide a local .graphql file path
+    rover subgraph publish my-graph@my-variant --name products --routing-url http://products-graphql.svc.cluster.local:4001/ --schema ./schema.graphql 
+
+    # Provide an introspection result via stdin
+    rover graph introspect http://localhost:4000 | rover subgraph publish my-graph@my-variant --name products --routing-url http://products-graphql.svc.cluster.local:4001/ --schema -
     ```
 
-| Name | Description  |
-|---|---|
-| `--endpoint` | <p>The URL of the running service to perform an introspection query on.</p><p>**Provide this only if the CLI should obtain your schema via introspection.**</p> |
-| `--localSchemaFile` | <p>The path of the schema file to register.</p><p>**Provide this only if the CLI should obtain your schema via a local file.**</p> |
-| `--key` | <p>The graph API key that the CLI should use to authenticate with the schema registry.</p><p>By default, the CLI uses the value of the `APOLLO_KEY` environment variable.</p> |
-| `--graph` | The name of your graph in Apollo Studio (e.g., `docs-example-graph`) |
-| `--variant` | <p>The [variant of your graph](https://www.apollographql.com/docs/studio/schema/registry/#managing-environments-with-variants) to register the schema with.</p><p>The default value is `current`.</p> |
-| `--serviceName` | The service's unique name in your federated graph (e.g., `products`). |
-| `--serviceURL` | The URL that your federated gateway will use to communicate with the service. |
-
-As you register your service schemas, the schema registry attempts to **compose** their latest versions into a single **federated schema**. Whenever composition succeeds, your gateway can fetch the latest federated schema from the registry.
+As you register your subgraph schemas, the schema registry attempts to **compose** their latest versions into a single **supergraph schema**. Whenever composition succeeds, your gateway can fetch the latest supergraph schema from the registry.
```

Comment:
Coming soon is `rover supergraph fetch`, which we _might_ want to mention here. We can also tell them they can then view their supergraph in the Schema->SDL tab in Studio.

Reply:
Will this command be included in 1.0?

Reply:
it'll be in the next release we cut! i think we're going for 0.1.0 instead of 1.0.0 but, yes!